src/app/components/work-item-link/work-item-link.component.less

```diff
@@ -11,14 +11,14 @@
   }
   .link-header-nav-list {
     border-top: 1px solid;
-    border-top-color: $color-pf-black-200;
+    border-top-color: @color-pf-black-200;
     li {
       &.toggle-icon {
         width: 40px;
         padding-top: 9px;
         padding-bottom: 9px;
-        @extend .dib;
-        @extend .textCenter;
+        &:extend(.dib);
+        &:extend(.textCenter);
```

Comment:
why not just use
```less
display: inline-block;
text-align: center;
```
instead of
```less
&:extend(.dib);
&:extend(.textCenter);
```

Using the full markup makes it easier to read and find on errors.